Formula/logstash.rb

```diff
@@ -1,29 +1,26 @@
 class Logstash < Formula
   desc "Tool for managing events and logs"
   homepage "https://www.elastic.co/products/logstash"
-  url "https://artifacts.elastic.co/downloads/logstash/logstash-oss-7.10.1-darwin-x86_64.tar.gz"
-  sha256 "8e5b8d2557029de9278361a8bb5ef86144a0d2176bb9c586a4170434e7ba15db"
+  url "https://github.com/elastic/logstash/archive/v7.10.1.tar.gz"
+  sha256 "9f4732d3c324d27ed348060eccf38840ec74f6baf155ef5f7347346b714a1c58"
   license "Apache-2.0"
+  version_scheme 1
   head "https://github.com/elastic/logstash.git"
 
   livecheck do
     url :head
     regex(/^v?(\d+(?:\.\d+)+)$/i)
   end
 
-  bottle :unneeded
-
   depends_on "openjdk@11"
 
   def install
-    if build.head?
-      # Build the package from source
-      system "rake", "artifact:tar"
-      # Extract the package to the current directory
-      mkdir "tar"
-      system "tar", "--strip-components=1", "-xf", Dir["build/logstash-*.tar.gz"].first, "-C", "tar"
-      cd "tar"
-    end
+    # Build the package from source
```

Comment:
```suggestion
    # remove non open source files
    rm_rf "x-pack"

    # Build the package from source
```

As we do with all elastic formulae